client/src/components/graph/graph.tsx

```diff
@@ -595,7 +595,13 @@ class Graph extends React.Component<GraphProps, GraphState> {
       const graphImageURI = offscreenCanvas.toDataURL("image/png");
 
       if (mountCapture) {
-        this.setState({ testImageSrc: graphImageURI });
+        this.setState(({ testImageSrc }) => ({
+          /**
+           * (thuang): We want to remove the test image if there's already one
+           */
+          testImageSrc: testImageSrc ? null : graphImageURI,
```

Comment:
this is needed to ensure we can remove the mounted image if we make further canvas updates